### DISCUSSION TITLE
ReCaptcha problems

### DISCUSSION THREAD

**Discussion by akajhon** [intelowlproject/GreedyBear#316] [Q&A] [answered]
Hello team!

Could someone help me with the ReCaptcha configuration? How could I completely disable the Captcha's?

Thanks in advance!

**Comment by mlodic** [comment]
hey, recaptcha is disabled by default (see: https://greedybear.readthedocs.io/en/latest/Usage.html#recaptcha-configuration).
did you find any problem so we can replicate it? ty

**Comment by akajhon** [comment]
Hello, @mlodic !

When freshly installing Greedy Bear (GB), I encounter the following message when logging in with the superuser:

![image](https://github.com/user-attachments/assets/cb5600db-1ea0-4054-8876-d093e8877c6e)

The default configurations I have, after cloning the repo for the first time are:

**docker/env_file :**

```plaintext
# drf-recaptcha
## localhost, internal deployments
RECAPTCHA_SECRET_KEY_GB_LOCAL=""
## public deployment
RECAPTCHA_SECRET_KEY_GB_PUBLIC=""
```

**frontend/public/env.js :**

```plaintext
window.$env = {
  // Choose one of the below,
  // a) localhost, internal deployments
  RECAPTCHA_SITEKEY: "",
  // b) public deployment
  // RECAPTCHA_SITEKEY: "",
};
```

Are these configurations correct? Is there something I am missing or need to do?

Thanks for your help!

**Reply by akajhon** [reply]
Just to mention... running it on a Ubuntu 22.04 Live server!

**Reply by mlodic** [reply]
we noticed that some new frontend dependency broke the support that we have for recaptcha. We are going to remove in the upcoming version cause we have no time to actually solve it

**Reply by mlodic** [reply, marked as answer]
try again with the new release please

**Reply by akajhon** [reply]
It worked out for me! 😁 Problem solved in the New Release!

Although, I had to add the lines below in the Dockerfile for it to be built.

```
RUN sed -i 's/alpine\.gliderlabs\.com/dl-cdn.alpinelinux.org/g' /etc/apk/repositories
RUN sed -i '/^@edge/d' /etc/apk/repositories
RUN apk update
```
Reference:

[APK Not Working on Alpine Linux](https://forums.docker.com/t/apk-not-working-on-alpine-linux-image/140574/2)

**Reply by mlodic** [reply]
cool, if you have any suggestions or feedback about the project, feel free to share. Do you have a cluster of tpots?
